src/data/index.js

```diff
@@ -24,7 +24,10 @@ export const elements = [
   },
 ];
 
-const genText = () => Math.random().toString(36).substring(7);
+const genText = () =>
+  Math.random()
+    .toString(36)
```

Comment:
Dont like these magic numbers...should be put into own constants

Reply:
Please submit a pull request :) 👍

Reply:
Dont know what those numbers are........

Reply:
Creating an issue for this